demos/CMakeLists.txt

```
@@ -47,3 +47,10 @@ if((${CMAKE_SYSTEM_NAME} MATCHES "^(Linux)$") AND
   add_executable(meltdown meltdown.cc)
   target_link_libraries(meltdown safeside)
 endif()
+
+if((${CMAKE_SYSTEM_NAME} MATCHES "^(Linux)$") AND
+    (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^aarch64$"))
```

```suggestion
   (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^(aarch64)$"))
```

as in #36, also fixing alignment